<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="38657" started-at="2020-05-13T02:23:38Z" finished-at="2020-05-13T02:24:17Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="38657" started-at="2020-05-13T02:23:38Z" finished-at="2020-05-13T02:24:17Z">
      <class name="cn.huida.test.testAssert">
        <test-method status="PASS" signature="before()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="before" is-config="true" duration-ms="5094" started-at="2020-05-13T10:23:38Z" finished-at="2020-05-13T10:23:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="FAIL" signature="test1()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="test1" duration-ms="3182" started-at="2020-05-13T10:23:43Z" finished-at="2020-05-13T10:23:46Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[惠达Title[惠达精准作业平台登录]和百度Title[百度一下，你就知道]不一样]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 惠达Title[惠达精准作业平台登录]和百度Title[百度一下，你就知道]不一样
	at org.testng.Assert.fail(Assert.java:94)
	at cn.huida.test.testAssert.test1(testAssert.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="after()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="after" is-config="true" duration-ms="5906" started-at="2020-05-13T10:23:46Z" finished-at="2020-05-13T10:23:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
        <test-method status="PASS" signature="before()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="before" is-config="true" duration-ms="4213" started-at="2020-05-13T10:23:52Z" finished-at="2020-05-13T10:23:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="PASS" signature="test2()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="test2" duration-ms="1633" started-at="2020-05-13T10:23:56Z" finished-at="2020-05-13T10:23:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="PASS" signature="after()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="after" is-config="true" duration-ms="5715" started-at="2020-05-13T10:23:58Z" finished-at="2020-05-13T10:24:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
        <test-method status="PASS" signature="before()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="before" is-config="true" duration-ms="4515" started-at="2020-05-13T10:24:04Z" finished-at="2020-05-13T10:24:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="PASS" signature="test3()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="test3" duration-ms="2526" started-at="2020-05-13T10:24:08Z" finished-at="2020-05-13T10:24:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method status="PASS" signature="after()[pri:0, instance:cn.huida.test.testAssert@4883b407]" name="after" is-config="true" duration-ms="5848" started-at="2020-05-13T10:24:11Z" finished-at="2020-05-13T10:24:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- cn.huida.test.testAssert -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
